ui/pages/Project/components/CreateVariantButton.jsx

```diff
@@ -205,7 +206,7 @@ const SV_FIELDS = [
       max: 12,
     },
   },
-].map(formatField)
+].map(formatField).map(field => (field.validate ? { ...field, label: `${field.label}*` } : field))
```

Comment:
this looks like its copy/pasted from above so you could move it into the shared `formatField` helper